ruby/vampires.rb

```diff
@@ -0,0 +1,37 @@
+
+puts "How many employees are being processed?"
+number_of_employees_to_process = gets.chomp
+until number_of_employees_to_process == 0
+	puts "What is your name?"
+	employee_name = gets.chomp
+	puts "How old are you?" 
+	employee_age = gets.chomp
+	puts "What year were you born?"
+	employee_birth_year = gets.chomp
+	puts "Our company cafeteria serves garlic bread. Should we order some for you?"
+	want_garlic_bread = gets.chomp
+	puts "Would you like to enroll in the company health insurance?"
+	want_health_insurance = gets.chomp
+	allergy = nil
+	result = "Results inconclusive."
+	until (allergy == "done") || (allergy == "sunshine")
+		puts "Name any allergies one at a time please. If there are no more to list, please reply done."
+		allergy = gets.chomp
+		if allergy == "sunshine"
+			result = "Probably a vampire."
+	end
+
+	if ((employee_age.to_i == Date.today.year - employee_birth_year.to_i) || (employee_age.to_i - 1 == Date.today.year - employee_birth_year.to_i)) && ((want_garlic_bread=="yes") || (want_health_insurance=="yes"))
+		result = "Probably not a vampire"
+	if ((employee_age.to_i != Date.today.year - employee_birth_year.to_i)|| (employee_age.to_i - 1 == Date.today.year - employee_birth_year.to_i)) && ((want_garlic_bread=="no") || (want_health_insurance=="no"))
```

Comment:
You wrote in multiple if statements but an if/elsif/else flow works in the way that the first conditional that's evaluated to true will break the flow but we want to keep them in order as the challenge said. If we are to follow the constraints we need to be more creative. What if we made each individual scenario it's own respective if block? What would go inside each if block? Surely we can't just puts out the result otherwise our screen would be full of each vampire check that passed. What we could do is keep setting a variable result within each if block and then at the end of all the blocks display the result to look something like this
```
if condition_1
	result = 'low'
end

if condition_2
	result = 'medium'
end

if condition_3
	result = 'high'
end

puts result
```
Your code was very close, you didn't `end` each if statement though.